src/pretix/presale/templates/pretixpresale/event/checkout_addons.html

```diff
@@ -57,16 +57,28 @@ <h3 class="panel-title">
         </div>
 
         {% if cross_selling_data %}
-            <details class="panel panel-default cross-selling" open>
+            {% if forms %}
+                <details class="panel panel-default cross-selling" open>
+            {% else %}
+                <div class="panel panel-default cross-selling">
```

Comment:
Hm, just stumbled upon this. It creates invalid HTML in combination with the summary element. Not that it’s super bad, but not ideal for a11y. So could you move the whole summary-block with headline „more recomandations“ into the if-block and in the else-block replace the summary-tag with a div.